mkdocs2sphinx/convert_files.py

```diff
@@ -101,21 +100,23 @@ def convert_files(path, block_list, replacement_map, license_text, files_no_lice
 
     for root, dirs, files in os.walk(path):  # pylint: disable=unused-variable
```

Comment:
```suggestion
    for p in Path(path).glob('**/*'): 
```

?

Reply:
you can get rid of `import os` with this

Reply:
If there was a pathilb company, Tom would be their ambassador.

Reply:
https://i.redd.it/scc7uyeonxi11.png